docs/Changelog.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## In version 1.19.0 (Not released yet)
```

Comment:
Take a look again how the section for each version looks like :) It looks slightly different

Reply:
Uhh, still it is done otherwise then in rest of document. Take a look again please.

Reply:
Fortunately someone was first, and now I can use his changes in Changelog ;)